sensor.py

```diff
@@ -487,8 +514,24 @@
                                     }
                                 }
                             },
+                            "Supply_Air_Temperature_Sensor": {
+                                "tags": [ TAG.Sensor, TAG.Temperature, TAG.Air, TAG.Supply ],
+                                "substances": [ [ BRICK.measures, BRICK.Temperature ], [ BRICK.measures, BRICK.Supply_Air ], ],
+                                "subclasses": {
+                                    "Cooling_Coil_Supply_Air_Temperature_Sensor": {
+                                        "tags": [ TAG.Cooling, TAG.Coil, TAG.Supply, TAG.Air, TAG.Temperature, TAG.Sensor ],
+                                    },
+                                    "Heat_Wheel_Supply_Air_Temperature_Sensor": {
```

Comment:
I think we decided to remove equipment tag from point names.

Reply:
Good catch! I removed these plus a few others